pkg/snapshot/restorer/restorer_test.go

```diff
@@ -1067,6 +1067,129 @@ var _ = Describe("Running Restorer when both v1 and v2 directory structures are
 	})
 })
 
+var _ = Describe("Running Restorer when base snapshot has etcd auth enabled", func() {
+	var (
+		restorer            *Restorer
+		restorePeerURLs     []string
+		clusterUrlsMap      types.URLsMap
+		peerUrls            types.URLs
+		baseSnapshot        *brtypes.Snapshot
+		deltaSnapList       brtypes.SnapList
+		store               brtypes.SnapStore
+		deltaSnapshotPeriod time.Duration
+		ep                  []string
+		etcdDataDir         string
+		restoreTempDir      string
+		testDir             string
+		storeDir            string
+		restorationConfig   *brtypes.RestorationConfig
+	)
+	const (
+		restoreName             string = "default"
+		restoreClusterToken     string = "etcd-cluster"
+		restoreCluster          string = "default=http://localhost:2380"
+		skipHashCheck           bool   = false
+		maxFetchers             uint   = 6
+		maxCallSendMsgSize             = 2 * 1024 * 1024 //2Mib
+		maxRequestBytes                = 2 * 1024 * 1024 //2Mib
+		maxTxnOps                      = 2 * 1024
+		embeddedEtcdQuotaBytes  int64  = 8 * 1024 * 1024 * 1024
+		autoCompactionMode      string = "periodic"
+		autoCompactionRetention string = "0"
+		embeddedEtcdPortNo      string = "9089"
+		rootUsername                   = "root"
+		rootPassword                   = "toor"
```

Comment:
I'm also not quite sure about how to avoid gitguradian detection of password, may be you can try with wrong spelling of password eg: `psswrd` 🤷